ath.sh

```diff
@@ -8,6 +8,12 @@ cd "$(dirname "$0")"
 # https://github.com/jenkinsci/acceptance-test-harness/releases
 export ATH_VERSION=5541.va_c0a_36b_b_f825
 
+if [[ $# -eq 0 ]]; then
+	export BROWSER=firefox
+else
+	export BROWSER=$1
+fi
+
```

Comment:
The changes in this file are just plumbing through the first argument as the browser to ATH to support Launchable flavors as described in the previous review comment.